network-api/networkapi/wagtailpages/migrations/0019_auto_20201208_2004.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.16 on 2020-12-08 20:04
+
+from django.db import migrations
+import networkapi.wagtailpages.pagemodels.customblocks.articles
+import wagtail.contrib.table_block.blocks
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0018_auto_20201126_2017'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='articlepage',
+            name='body',
+            field=wagtail.core.fields.StreamField([('content', networkapi.wagtailpages.pagemodels.customblocks.articles.ArticleRichText(features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'link', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'])), ('callout', wagtail.core.blocks.BlockQuoteBlock(template='wagtailpages/blocks/article_blockquote_block.html')), ('table', wagtail.contrib.table_block.blocks.TableBlock(template='wagtailpages/blocks/article_table_block.html')), ('double_image', wagtail.core.blocks.StructBlock([('image_1', wagtail.images.blocks.ImageChooserBlock()), ('image_1_caption', wagtail.core.blocks.RichTextBlock(features=['italic', 'bold', 'link'], label='Image caption', required=False)), ('image_2', wagtail.images.blocks.ImageChooserBlock()), ('image_2_caption', wagtail.core.blocks.RichTextBlock(features=['italic', 'bold', 'link'], label='Image caption', required=False))])), ('full_width_image', wagtail.core.blocks.StructBlock([('image', wagtail.images.blocks.ImageChooserBlock()), ('image_height', wagtail.core.blocks.IntegerBlock(default=410, help_text='A custom height for this image. The image will be 1400px wide by this height. Note: This may cause images to look pixelated. If the browser is wider than 1400px the height will scale vertically while the width scales horizontally')), ('caption', wagtail.core.blocks.RichTextBlock(features=['italic', 'bold', 'link'], label='Image caption', required=False))]))]),
+        ),
```

Comment:
Stream blocks always result in a migration even though it's not technically required. There's a way around this if we want to get a bit hacky, otherwise, we're stuck with this.

Reply:
what's one more.